pcsx2/windows/VCprojects/pcsx2.vcxproj.filters

```diff
@@ -1060,9 +1066,6 @@
     <ClCompile Include="..\..\DEV9\smap.cpp">
       <Filter>System\Ps2\DEV9</Filter>
     </ClCompile>
-    <ClCompile Include="..\..\DEV9\net.cpp">
-      <Filter>System\DEV9</Filter>
-    </ClCompile>
```

Comment:
This change is going to conflict with a similar change I did in my HDD pr here https://github.com/PCSX2/pcsx2/pull/3932/commits/ea1e27de88dca2b998717f01545434f061498a20

Reply:
I'll deal with the rebase when your PR gets merged, not much I can do there. If by some miracle my PR ends up being merged before ping me and I'll rebase yours!

Reply:
Alright